pkg/planner/core/columnar_index_utils.go

```diff
@@ -66,3 +66,18 @@ func buildInvertedIndexExtra(
 		},
 	}
 }
+
+func buildTextInvertedIndexExtra(
```

Comment:
I believe "FTS Index" is a more accurate name than "text inverted index".

Reply:
This is about the planner side's scope.
Should we name it as fulltext search index if we are using this index to do a query like `col = string_const`

Reply:
Would that also use this method?
I think this util function can only `FTSQueryInfo` now. We can rename it at that time to avoid confusion.